src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const createDomNode = (type) => document.createElement(type);
+
+const addClassesToDomNode = (node, classes) => node.classList.add(...classes);
+
+const addTextToDomNode = (node, text) => (node.textContent = text);
+
+const appendNode = (parentNode, node) => parentNode.append(node);
+
+const createNotification = (type, parentNode, classes, text) => {
+  const node = createDomNode(type);
+
+  addClassesToDomNode(node, classes);
+
+  addTextToDomNode(node, text);
+
+  appendNode(parentNode, node);
+};
+
+const promise1 = new Promise((resolve) => resolve('Promise was resolved!'));
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
+});
+
+logo.addEventListener('click', () => {
+  promise1
+    .then((res) => {
+      createNotification('div', body, ['message'], res);
+    })
+    .catch((_error) => {
+      createNotification(
+        'div',
+        body,
+        ['message', 'error-message'],
+        _error.message,
+      );
+    });
+});
```

Comment:
The event listener for the `logo` click event is directly using the `promise1`. This is a bit of a code smell because it tightly couples the click event to the promise. Instead, you should use a function that returns a promise and call that function inside the event listener. This way, you can easily change the promise or the way it's called without having to change the event listener code.